workflow-controller/controller-service/src/main/java/io/hyscale/controller/commands/deploy/HyscaleDeployServiceCommand.java

```diff
@@ -112,44 +131,51 @@
     private DeployComponentInvoker deployComponentInvoker;
     
     @Autowired
-    private ServiceSpecMapper serviceSpecMapper;
-
+    private ManifestAndDeployHelper manifestAndDeployHelper;
+    
     @Override
     public Integer call() throws Exception {
         if (!CommandUtil.isInputValid(this)) {
             return ToolConstants.INVALID_INPUT_ERROR_CODE;
         }
+        HyscaleCommandSpec commandSpec = new HyscaleCommandSpec();
```

Comment:
HyscaleCommandSpec Builder would probably be better.

Reply:
Sure